src/dry-package.ts

```diff
@@ -1,65 +1,71 @@
-import * as fs from 'fs';
 import * as deepDiff from 'deep-diff';
-import {DryPackageContent} from './dry-package-content';
-import {JsonUtils} from './json-utils';
-import {DependencyResolver} from './dependency-resolver';
-import {NpmPackage} from './npm-package';
 import * as merge from 'deepmerge';
+import * as fs from 'fs';
+
+import { DependencyResolver } from './dependency-resolver';
+import { DryPackageContent } from './dry-package-content';
+import { JsonUtils } from './json-utils';
+import { NpmPackage } from './npm-package';
+
+// TODO: consider adding a type to 'any'
+// tslint:disable-next-line:no-any
+export type WeakDryPackageContent = DryPackageContent & any;
 
 /**
  * The dry package (i.e. package-dry.json) is the pendant of the npm package (i.e. package.json).
  * A dry package is an npm package added of a DryPackageDescriptor.
  */
 export class DryPackage {
-
-    private static readonly PACKAGE_DRY_JSON = 'package-dry.json';
-
-    private constructor(private readonly dependencyResolver: DependencyResolver,
-                        private readonly location: string,
-                        private _content: DryPackageContent & any) {
-
-    };
-
-    static readFromDisk(dependencyResolver: DependencyResolver): DryPackage {
+    public static readFromDisk(dependencyResolver: DependencyResolver): DryPackage {
```

Comment:
Sorry I didn't see that earlier.
Is it possible to have all attributes always at the top of the class and all methods come after?

Reply:
Done, overwritten the default `member-ordering` rule, which states that static functions have to be before constructor

I personally think everything `static` should come before the constructor, cos it is not instance specific, but again, its my opinion :P